lib/fetcher/fetcher_service.rb

```diff
@@ -0,0 +1,14 @@
+class FetcherService
+  def initialize(fetchers)
+    @fetchers = fetchers
+  end
+
+  def fetch(url)
+    @fetchers.each do |fetcher|
+      return fetcher.fetch(url)
+    rescue VersionNotSupportedError
+      # Ignore as we are logging this error elsewhere
```

Comment:
Are we? Isn't it more that we're happy the version not being supported, as others may be supported?